docs/scheduler/rescheduling.md

```diff
@@ -0,0 +1,46 @@
+<!--[metadata]>
++++
+title = "Docker Swarm recheduling"
+description = "Swarm rescheduling"
+keywords = ["docker, swarm, clustering, rescheduling"]
+[menu.main]
+parent="smn_workw_swarm"
+weight=5
++++
+<![end-metadata]-->
+
+# Rescheduling
+
+The Docker Swarm scheduler is able to detect node failure and
+restart it's container on another node.
+
+## Rescheduling policies
+
+The rescheduling policies are:
+
+* `on-node-faillure`
+* `off` (default if not specified)
+
+
+When you start a container, use the env var `reschedule` or the
+label `com.docker.swarm.reschedule-policy` to specify the policy to
+apply to the container.
+
+```
+# do not reschedule (default)
+$ docker run -d -e reschedule:off redis
+# or
+$ docker run -d -l com.docker.swarm.reschedule-policy=off redis
```

Comment:
It's the same with affinities and contraints:

1) in swarm < 0.3 we used env because we had only this.
2) in swarm > 0.4 we started storing them in labels because it's cleaner.
but we kept the `env` as compatibility.
3) after a few release, many people still use `env` because it's much more simpler to use.

Since it's stored in labels, there is no way to remove `-l com.docker.swarm.reschedule-policy=*`
we could remove `-e reschedule:*` but that's much more easier to use.
I'd like to keep both.

@abronan ?